docs/how-to/configure.md

```diff
@@ -78,11 +78,19 @@ The *Telemetry* switch enables gathering and sending basic **anonymous** data to
 
 The following information is gathered:
 
-- PMM Version;
-- Installation Method (Docker, AMI, OVF);
-- the Server Uptime;
-- Advisors (enabled or disabled);
-- Integrated Alerting Status (enabled or disabled).
+- PMM Server Integration Alerting feature enabled/disabled
+- PMM Server Security Thread Tool feature enabled/disabled
+- PMM Server Backup feature enabled/disabled
+- PMM Server DBaaS feature enabled/disabled
+- PMM Server Check Updates feature disabled
+- Detailed information about version and comment of monitored MySQL services
```

Comment:
I am unsure what this means "comment of monitored MySQL services"

Reply:
There are now two metrics version and "version comments" ... this is more extended description of the version, so we can see is this community or Percona version of MySQL

Reply:
Detailed information should contain everything ideally. " Detailed information about the version of monitored MySQL services"